ui/src/app/shared/components/tri-state-component.ts

```diff
@@ -27,7 +27,9 @@ export class Tristate implements AfterViewInit, DoCheck, OnInit  {
   constructor(private _changeDetectorRef: ChangeDetectorRef) { }
 
   private setState() {
-    if (!this._items) return;
+    if (!this._items) {
+      return;
+    }
     var count: number = 0;
```

Comment:
use let instead of var. I know this isn't a part of the PR, but just noticed it.